packages/plugin-window-onerror/onerror.js

```diff
@@ -20,11 +20,11 @@ module.exports = {
             report = new client.BugsnagReport('window.onerror', String(error), decorateStack(client.BugsnagReport.getStacktrace(error, 1), url, lineNo, charNo), handledState)
             report.updateMetaData('window onerror', { error })
           }
-        } else if ((typeof messageOrEvent === 'object' && messageOrEvent !== null) && !url && !lineNo && !charNo && !error) {
+        } else if ((typeof messageOrEvent === 'object' && messageOrEvent !== null) && (!url || typeof url !== 'string') && !lineNo && !charNo && !error) {
```

Comment:
The line length in this file is getting pretty long. Do we not have coding standards for that in the js repo? Should we use prettier or something?

This boolean logic is also pretty complex, and there is very little commenting in the file. Why do we just do `!url` and `messageOrEvent is !== null` explicitly?

Reply:
It seems like we should comment saying why we are checking urls type here (jquery). We probably also want to link to https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror

Reply:
It's certainly not a pretty file. We use [standard](https://standardjs.com/rules.html) for style rules which is not opinionated about line length.

I'll add some comments and see what I can do about the line length without having an adverse effect on output file size.